packages/_tests/fixtures.coffee

```diff
@@ -108,3 +132,24 @@ if Meteor.isAppTest
       catch error
         # this user shouldn't belong in the production database
         console.warn("TestingAdmin user '#{email}' exists")
+
+    addIncidents: (eventId, incidentCount) ->
+      Meteor.call 'addEventSource', testSource2, eventId, (error, articleId) ->
```

Comment:
Since these methods run on the server they can be called synchronously without a callback. Using async calls might cause timing issues in the tests since the addIncidents method will finish before the async calls complete.